pipeline/splitters.go

```diff
@@ -23,9 +23,14 @@ import (
 	"crypto/subtle"
 	"errors"
 	"fmt"
-	"github.com/mozilla-services/heka/message"
 	"hash"
 	"regexp"
+
+	"github.com/mozilla-services/heka/message"
+)
+
+const (
+	MULTILINE_SPLITTER_MAX_LINES = 99
```

Comment:
Yeah, agreed. I called it out in the description of the PR, but it's easy enough to make it configurable.